public/assets/css/ci-theme.css

```
@@ -1288,13 +1288,12 @@ section#content-outer-discover{
 
 
 .ci-version-boxes{
-
 	float:left;
-	width:31%;
-	min-height:700px; /*240px*/
+	width:100%;/*31%*/

```

No need to leave comments in place - `git` will track the whole history so cleaner just to remove things.
```suggestion
	width:100%;
```